roxie/udplib/udpsim.cpp

```diff
@@ -148,16 +163,35 @@ void initOptions(int argc, const char **argv)
     udpTraceLevel = options->getPropInt("@udpTraceLevel", 1);
     udpTraceTimeouts = options->getPropBool("@udpTraceTimeouts", true);
     udpResendLostPackets = options->getPropBool("@udpResendLostPackets", true);
-    udpRequestToSendTimeout = options->getPropInt("@udpRequestToSendTimeout", 1000);
-    udpRequestToSendAckTimeout = options->getPropInt("@udpRequestToSendAckTimeout", 1000);
+
+    udpPermitTimeout = options->getPropInt("@udpPermitTimeout", udpPermitTimeout);
+    udpRequestTimeout = options->getPropInt("@udpRequestTimeout", udpRequestTimeout);
+    udpFlowAckTimeout = options->getPropInt("@udpFlowAckTimeout", udpFlowAckTimeout);
+    updDataSendTimeout = options->getPropInt("@udpDataSendTimeout", updDataSendTimeout);
+    udpResendTimeout = options->getPropInt("@udpResendTimeout", udpResendTimeout);
+    udpMaxPermitDeadTimeouts = options->getPropInt("@udpMaxPermitDeadTimeouts", udpMaxPermitDeadTimeouts);
+    udpRequestDeadTimeout = options->getPropInt("@udpRequestDeadTimeout", udpRequestDeadTimeout);
+
+    udpAssumeSequential = options->getPropBool("@udpAssumeSequential", udpAssumeSequential);
+    udpAllowAsyncPermits = options->getPropBool("@udpAllowAsyncPermits", udpAllowAsyncPermits);
     udpMaxPendingPermits = options->getPropInt("@udpMaxPendingPermits", 1);
+    udpMinSlotsPerSender = options->getPropInt("@udpMinSlotsPerSender", udpMinSlotsPerSender);
+
     udpTraceFlow = options->getPropBool("@udpTraceFlow", false);
-    packetsPerThread = options->getPropInt("@packetsPerThread");
     udpTestUseUdpSockets = !options->getPropBool("@useQueue");
+    udpAdjustThreadPriorities = options->getPropBool("@udpAdjustThreadPriorities", udpAdjustThreadPriorities);
+    packetsPerThread = options->getPropInt("@packetsPerThread");
 
     isUdpTestMode = true;
-    roxiemem::setTotalMemoryLimit(false, false, false, 20*1024*1024, 0, NULL, NULL);
+    roxiemem::setTotalMemoryLimit(false, true, false, 20*1024*1024, 0, NULL, NULL);
```

Comment:
ok, why allowTransparentHugePages now ?

Reply:
Only that I saw that it was being set to false, and that isn't the default for roxie.  I doubt it makes any difference, but changed to make it consistent.